wrappers/SMath/coolprop_wrapper/install.bat.template

```diff
@@ -1,4 +1,4 @@
 
 set VERSION=${COOLPROP_VERSION}
-xcopy %VERSION% %AppData%\SMath\extensions\plugins\2be8d14b-ba62-4002-a959-9e5dad905310\%VERSION%\ /S
-xcopy config.ini %AppData%\SMath\extensions\plugins\2be8d14b-ba62-4002-a959-9e5dad905310\config.ini
\ No newline at end of file
+xcopy %VERSION% %AppData%\SMath\extensions\plugins\ca92ef03-c7da-4888-98ad-528482733e2f\%VERSION%\ /S
```

Comment:
Oh, that's because it's the GUID that is used in downstream SMath repo, and it's now much more widely used, so I think it's good to standardise on one. Otherwise SMath complains that they are incompatible